packages/@react-aria/numberfield/intl/ar-AE.json

```diff
@@ -1,5 +1,5 @@
 {
-  "decrease": "Decrease {fieldLabel}",
-  "increase": "Increase {fieldLabel}",
-  "Number field": "Number field"
+  "Number field": "حقل رقمي",
```

Comment:
This shouldn't have been sentence cased. Can we make them all lower cased "number field"? Is it easier for you to do that in this pr?

Reply:
I think this needs a change in .ts file as well, it's currently....
```
'aria-roledescription': (!isIOS() ? formatMessage('Number field') : null),
```

Change to?
```
'aria-roledescription': (!isIOS() ? formatMessage('numberField') : null),
```
and
```
 "numberField": "حقل رقمي",
```

I can do that in this PR if sounds okay,

Reply:
Yes `numberField` works. Happy to have it done in this pr if you have time.

Reply:
Hi @dannify I've made the change for English and localized files, PR is updated.